gitops/gitops_test.go

```diff
@@ -143,10 +143,10 @@ var _ = Describe("Gitops functions for managing ApplicationSnapshots", Ordered,
 		Expect(gitops.HaveHACBSTestsSucceeded(hasSnapshot)).To(BeTrue())
```

Comment:
It won't let me comment on it for some reason, but line 141 `s/applicationsnapshot/snapshot/`

Reply:
Ack, fixed